setup_undercloud.yml

```diff
@@ -57,6 +57,7 @@
           delegate_to: "{{ hammer_host }}"
           vars:
             ansible_user: root
+            ansible_python_interpreter: /usr/bin/python2.7
```

Comment:
Why do you need /usr/bin/python2.7 specifically? Doesn't /usr/bin/python not work?

Reply:
Yeah, I tested it /usr/bin/python and it works. Thanks have updated it.